### DISCUSSION TITLE
Receive Neo-tree warn: invalid mapping for Y : nil

### DISCUSSION THREAD

**Discussion by stimw** [LazyVim/LazyVim#2459] [Q&A] [answered]
<img width="1151" alt="image" src="https://github.com/LazyVim/LazyVim/assets/30977647/04ea7df6-0e2c-490c-a51a-f54b3a633c9b">

always receive this warn when open the git explorer...not sure why.

Neovim v0.9.5 on MacOS

fresh new installation of the latest version, just change the colorscheme add some extras:

```json
"extras": [
    "lazyvim.plugins.extras.editor.leap",
    "lazyvim.plugins.extras.lang.json",
    "lazyvim.plugins.extras.lang.typescript",
    "lazyvim.plugins.extras.vscode"
  ],
 ```

I have tried many times to delete the cache and reinstall, but I will still receive this warn.

**Comment by dpetka2001** [comment]
This comes from a recent added keymap in Neo-tree. It seems that when there are no nodes in Neo-tree (like when you have a clean working tree in Git explorer of neo-tree) it gives this warning. Nothing to worry about.

**Reply by stimw** [reply]
Thank you for your reply.

No matter whether there is a node in neo-tree or not, this warning will be displayed...like in git explorer or in buffer explorer. 

And it's reproduced on a docker container using:

```bash
docker run -w /root -it --rm alpine:edge sh -uelic '
  apk add git lazygit neovim ripgrep alpine-sdk --update
  git clone https://github.com/LazyVim/starter ~/.config/nvim
  cd ~/.config/nvim
  nvim
'
```

update:

This really doesn't have any effect, but it's a little uncomfortable to have a high frequency.

**Reply by dpetka2001** [reply, marked as answer]
There's a fix for this [here](https://github.com/LazyVim/LazyVim/commit/eccf3b5e68035516c879b3e2b4d6bf8512c87976) scheduled to be merged in the next release. If you'd like to avoid seeing the error in the meantime, you could always just follow the latest `main` instead of release according to the [docs](https://www.lazyvim.org/configuration/lazy.nvim). The problem doesn't exist in the latest `main`.
